packages/cli/src/commands/dbCommands/generate.js

```diff
@@ -29,9 +35,13 @@ export const handler = async ({ verbose = true, force = true }) => {
   return await runCommandTask(
     [
       {
-        title: 'Generating the Prisma client...',
-        cmd: 'yarn prisma',
-        args: ['generate'],
+        title: schemaExists
+          ? 'Generating the Prisma client...'
+          : 'Skipping Prisma Client: no schema.prisma found',
+        cmd: schemaExists
+          ? 'yarn prisma'
+          : 'echo "no schema.prisma file found"',
+        args: schemaExists ? ['generate'] : [],
```

Comment:
I don't think `args` is always required, and moving it up here will make it easier to read.
```suggestion
        cmd: schemaExists
          ? 'yarn prisma generate'
          : 'echo "no schema.prisma file found"'
```

Reply:
👍